src/Sulu/Bundle/SnippetBundle/Admin/SnippetAdmin.php

```diff
@@ -114,6 +114,7 @@ function(Localization $localization) {
             (new Route('sulu_snippet.form.taxonomies', '/taxonomies', 'sulu_admin.list'))
                 ->addOption('resourceKey', 'snippets')
                 ->addOption('tabTitle', 'sulu_snippet.taxonomies')
+                ->addOption('adapters', ['table'])
```

Comment:
This makes the PHP tests fail 🙈 Sorry, didn't think about that. You should add it to [this assertion](https://github.com/sulu/sulu/blob/develop/src/Sulu/Bundle/SnippetBundle/Tests/Unit/Admin/SnippetAdminTest.php#L89).